src/Confluent.Kafka/Producer.cs

```diff
@@ -302,16 +303,17 @@ public Producer(IEnumerable<KeyValuePair<string, object>> config, bool manualPol
         ///     Initializes a new Producer instance.
         /// </summary>
         /// <param name="config">
-        ///     librdkafka configuration parameters (refer to https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md)
-        ///     topic configuration parameters should be specified via the "default.topic.config" config parameter.
+        ///     librdkafka configuration parameters (refer to https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md).
+        ///     Topic configuration parameters should be specified via the "default.topic.config" config parameter.
         /// </param>
         public Producer(IEnumerable<KeyValuePair<string, object>> config)
             : this(config, false, false) {}
 
 
         /// <summary>
-        ///     Poll for callback events. Only use this method on producer
-        ///     instances where background polling has been disabled.
+        ///     Poll for callback events. You will not typically need
```

Comment:
This hasn't changed - it's just clarifying. Background polling has always been enabled by default on the producer and i was thinking disabling it is an advanced use case. 

I guess you might want to poll from multiple threads to process delivery reports in parallel.. but how often is anyone going to want to do this? is it often going to be a bottleneck? I hadn't thought this through before. I think it would all work ok with the current API. we should sync up about this to discuss.